src/Configuration/Config/src/ChainedBuilderExtensions.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.Extensions.Configuration
 {
     /// <summary>
-    /// IConfigurationBuilder extension methods for the chaind configuration provider.
+    /// Extension methods for adding <see cref="IConfiguration"/> to a <see cref="IConfigurationBuilder"/>.
```

Comment:
Consider changing `a` to `an` (because it's spoken as `an I Configuration Builder).